Test.C

```diff
@@ -235,6 +235,57 @@ int main(){
         // Selling equipment
         // decrease amount in index amount if things are sold
         // increase credit if things are sold
+        // So this is a lot of pseudo code or not finished but idk how to do it fully but it a good framework 
+        // assumes that you can sell the equipment for the same price that is bought 
+        int equipSellSelect;
+        int equipSellAmount;
+        int equipSellTransaction;
+        int equipSellAnswer;
+        
+        printf("|----------------------------- Equipment Selling ------------------------------|\n");
+        printf("Equipment that you own:\n");
+        for(int i=0; i<NUM_EQUIPMENT; i++){
+            printf("%s %s %d %d\n", equipmentStock[i].name, equipmentStock[i].description, equipmentStock[i].cost, equipmentStock[i].messageAmount);
+            printf("Amount of %s that is in your inventory: %d\n", equipmentStock[i].name, player.equipmentInventory[i]);
+        }
+        printf("Your current credits are: %d\n", player.currentCredits); // current credit
+        printf("Please select the equipment you would like to sell or select %d to pass: ", NUM_EQUIPMENT + 1); // chose 11 becuase there are 10 options rn
+        scanf("\n%d", &equipSellSelect);
```

Comment:
No newlines in scanf
```suggestion
        scanf("%d", &equipSellSelect);
```